app/src/main/java/com/cornellappdev/volume/ui/screens/PublicationsScreen.kt

```diff
@@ -1,118 +1,141 @@
-    }
+                   Spacer(modifier = Modifier.height(10.dp))
+               }
+               item {
+                   when (val followingPublicationsState = publicationsUiState.followedPublicationsState) {
+                       PublicationsRetrievalState.Loading -> {
+                           Column(
+                               modifier = Modifier.fillMaxWidth(),
+                               horizontalAlignment = Alignment.CenterHorizontally
+                           ) {
+                               CircularProgressIndicator(color = VolumeOrange)
+                           }
+                       }
+                       PublicationsRetrievalState.Error -> {
+                           // TODO Prompt to try again, queryFollowingPublications manually (it's public). Could be that internet is down.
+                       }
+                       is PublicationsRetrievalState.Success -> {
+                           LazyRow(horizontalArrangement = Arrangement.spacedBy(24.dp)) {
 
-    val data =
-        navController.currentBackStackEntry?.savedStateHandle?.getLiveData<String>("fromIndividualPublication")
-            ?.observeAsState()
+                               items(followingPublicationsState.publications) { publication ->
+                                   CreateFollowPublicationRow(publication) {
+                                       onPublicationClick(publication)
+                                   }
+                               }
+                           }
+                       }
+                   }
+               }
+               item {
+                   Text(
+                       modifier = Modifier.padding(top = 30.dp),
+                       text = "More Publications",
+                       fontFamily = notoserif,
+                       fontSize = 20.sp,
+                       fontWeight = FontWeight.Medium
+                   )
 
+                   Spacer(modifier = Modifier.height(22.dp))
+               }
+               item {
+                   when (val unfollowingPublicationsState =
+                       publicationsUiState.morePublicationsState) {
+                       PublicationsRetrievalState.Loading -> {
+                           Column(
+                               modifier = Modifier.fillMaxWidth(),
+                               horizontalAlignment = Alignment.CenterHorizontally
+                           ) {
+                               CircularProgressIndicator(color = VolumeOrange)
+                           }
+                       }
+                       PublicationsRetrievalState.Error -> {
+                           // TODO Prompt to try again, queryFollowingPublications manually (it's public). Could be that internet is down.
+                       }
+                       is PublicationsRetrievalState.Success -> {
+                           LazyColumn(verticalArrangement = Arrangement.spacedBy(24.dp), modifier = Modifier
```

Comment:
Could this be changed to just be a column that shows publications at once instead of having to scroll? I highlighted how this could be done for another screen but it can be adapted for this as well! https://github.com/cuappdev/volume-compose-android/pull/10#discussion_r979094689